### PR TITLE
disable party indicators when sort is applied to table issue #1280

### LINKED ISSUE #1280: Player groups in match statistics
When sorting is applied, player group bindings acting strange:

Maybe need to disable sorting like on Dotabuff?

### PR DESCRIPTION
**Pull request by Argn0**
fixes #1280 

https://deploy-preview-1353--opendota-stage.netlify.com/matches/3582800334

### CODE REVIEW

File: src/components/Table/Table.jsx [hunk 1]
```diff
@@ -132,6 +132,10 @@ class Table extends React.Component {
     if (paginated) {
       data = data.slice(currentPage * pageLength, (currentPage + 1) * pageLength);
     }
+    data.forEach((row) => {
+      const r = row;
+      r.sortApplied = !!sortField && (sortField !== 'player_slot' || sortState !== 'asc');
```
Comment (howardchung) [inline src/components/Table/Table.jsx]: why the double negate?

Comment (Argn0) [reply src/components/Table/Table.jsx]: I use it instead of a type cast to boolean

Comment (howardchung) [reply src/components/Table/Table.jsx]: I prefer the type cast, it's more readable

Comment (Argn0) [reply src/components/Table/Table.jsx]: No idea what I was trying to do there, but it works just fine like this now. :)

**Pull request closed without merging**